tracer/src/Datadog.Trace/AppSec/Waf/IWaf.cs

```diff
@@ -21,15 +21,8 @@ internal interface IWaf : IDisposable
 
         internal DDWAF_RET_CODE Run(IntPtr contextHandle, IntPtr rawArgs, ref DdwafResultStruct retNative, ulong timeoutMicroSeconds);
 
-        /// <summary>
-        /// only this one returns a rulsetinfo from the waf
-        /// </summary>
-        /// <param name="rules">json rules</param>
-        /// <returns>returns InitOrUpdateResult</returns>
-        UpdateResult UpdateRules(string rules);
+        UpdateResult Update(IDictionary<string, object> arguments);
```

Comment:
Is this required on the interface? The only place I see it being used is in the Waf class itself

Reply:
removed in 3a2bc9d2bd36dee143c9feb6f50a721a36309a22